andela/annie-hall-rc
# #154440552 Implement Products Load Upon Scroll

#### What does this PR do?
Displays more products from database upon scroll by the user

#### Description of Task to be completed?
 * Write functions to display a limited number of products upon user scroll

#### How should this be manually tested?
Have the app working by:
 * run the command `reaction` on the command line to start the server
 * once app is running, navigate to the app using `localhost:3000`
* Scroll through the product list and you shall observe that products gets loaded automatically upon scroll

 #### Any background context you want to provide?
 Nil

 #### What are the relevant pivotal tracker stories?
#154440552

 #### Screenshots (if appropriate)
<img width="1440" alt="screen shot 2018-01-23 at 1 48 36 pm" src="https://user-images.githubusercontent.com/30434186/35276963-7a78b492-0045-11e8-974d-dd68faf2d4a6.png">
